src/main/java/com/worldwarofants/game/module/tutorial/TutorialView.java

```diff
@@ -12,12 +13,14 @@
 	private static final String RETREAT_DESC = "Retreat";
 	private static final String QUIT_DESC = "Quit";
 	
+	
```

Comment:
What's the point of this change? :D Look out for things like this. Another empty line is not necessary here.

Reply:
I've removed this errant new line.